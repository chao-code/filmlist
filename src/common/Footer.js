const Footer = () => (
  <footer className="container">
    <hr />
    <p className="text-center my-3">
      <small>Â© 2018. Powered by <span role="img" aria-label="Poop">ğŸ’©ğŸ’©</span></small>
    </p>
  </footer>
)

export default Footer
