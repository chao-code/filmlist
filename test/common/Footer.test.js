import { shallow } from 'enzyme'
import Footer from '../../src/common/Footer'

describe('<Footer />', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Footer />)
    expect(wrapper).toMatchSnapshot()
  })
})
