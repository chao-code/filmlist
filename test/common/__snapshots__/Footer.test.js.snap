// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render correctly 1`] = `
<footer
  className="container"
>
  <hr />
  <p
    className="text-center my-3"
  >
    <small>
      Â© 2018. Powered by 
      <span
        aria-label="Poop"
        role="img"
      >
        ðŸ’©ðŸ’©
      </span>
    </small>
  </p>
</footer>
`;
